src/libp2p/Libp2p.Protocols.Pubsub/Settings.cs

```diff
@@ -24,13 +25,7 @@ public class Settings
     public SignaturePolicy DefaultSignaturePolicy { get; set; } = SignaturePolicy.StrictSign;
     public int MaxIdontwantMessages { get; set; } = 50;
 
-    public Func<Message, string> GetMessageIdFunction = GetMessageId;
-
-    private static string GetMessageId(Message message)
-    {
-        Span<byte> bytes = new byte[message.From.Length + message.Seqno.Length];
-        return "";
-    }
+    public Func<Message, MessageId> GetMessageId = (Message message) => new MessageId(message.From.Concat(message.Seqno).ToArray());
```

Comment:
Isn't it better to convert this to a property like other members?

```suggestion
    public Func<Message, MessageId> GetMessageId { get; set; } = (Message message) => new MessageId([.. message.From, .. message.Seqno]);
```